src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -35,6 +35,10 @@
     <SignAssembly>true</SignAssembly>
     <DelaySign>true</DelaySign>
     <DefineConstants>$(DefineConstants);CORECLR;_USE_NLS_PLUS_TABLE;RESOURCE_SATELLITE_CONFIG;CODE_ANALYSIS_BASELINE</DefineConstants>
+    <!-- We don't use any of MSBuild's resolution logic for resolving the framework, so just set these two properties to any folder that exists to skip
```

Comment:
Would you mind opening an issue in BuildTools about this and linking to it here in the comments. Thanks.